### DISCUSSION TITLE
Make media folders visible, when selecting image

### DISCUSSION THREAD

**Discussion by martinptacek** [directus/directus#13452] [Feature Requests]
Directus supports folders in media storage. When I want to attach file, I see all the media together. That makes the management difficult with larger amount of pictures.

**Comment by rijkvanzanten** [comment]
~~Heya! Thank you for taking the time to submit this request!~~

~~It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:~~

~~This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!~~

~~_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._~~

This will launch very soon in https://github.com/directus/directus/pull/18320 🙂

**Reply by LasseRosenow** [reply]
Isn't this being implemented in #18320?

**Reply by rijkvanzanten** [reply]
Ah yes yes! I tried making sure I moved the active/implemented ones over properly. I've gone over 200+ feature requests so far, so they all blend together a bit 😵
